common_docs/EKS_SPECIFICS.md

```diff
@@ -1,10 +1,10 @@
 # EKS-Specific Info and Errata
 
-At Cribl, we develop our charts on our AWS EKS environments, but try to make sure that any implementation-specific configuration is done in annotations, etc. This document is intended to share our experience with our charts on the EKS platform. 
+At Cribl, we develop our charts on our AWS EKS environments, but we use annotations, etc. for implementation-specific configurations. In this document we share our experience with our charts on the EKS platform. 
```

Comment:
> we use annotations, etc. for implementation-specific
This reads weird. Do we need that etc.? Or perhaps it should be e.g.?

Reply:
I think it's fine in its current state. (The legacy sentence already wins an award for packing the most "-tion" verby nouns into the tightest space.) 😄